lib/config/options/index.ts

```diff
@@ -1214,6 +1214,13 @@ const options: RenovateOptions[] = [
     type: 'string',
     default: `renovate/`,
   },
+  {
+    name: 'branchPrefixOld',
+    description: 'Old Prefix to check for existing PRs.',
+    stage: 'branch',
+    type: 'string',
+    default: `renovate/`,
```

Comment:
This is a clever idea to automatically solve the problem for the majority of users who would only change the branchPrefix once?

Reply:
dont know about clever, it just seemed logical. As this flag is basically just a placeholder for the previous value of `branchPrefix`.